nextjs/pages/components/WebcamCapture.tsx

```diff
@@ -79,15 +79,18 @@ export default function WebcamCapture({ isCapturing, setIsCapturing, hideWebcam,
                     window.alert("You did the sign correctly!")
                 } else {
 
-                    let alertString: string = `Incorrect, recognised sign ${predictObject.prediction}\n`;
-                    const predictions = predictObject.predictions.sort((a, b) => (a.certainty > b.certainty ? -1 : 1));
+                    let alertString: string = `Incorrect, recognised sign ${predictObject.prediction} and not ${signLabel}\n`;
+                    const predictions = predictObject.allPredictions.sort((a, b) => (a.certainty > b.certainty ? -1 : 1));
 
+                    // Build multiline alert string with. Skip first because it is .prediction
```

Comment:
because it is .prediction? den forstår jeg ikke helt

Reply:
predictionObject.prediction

Reply:
Forstår det stadig ikke

Reply:
https://www.youtube.com/watch?v=OCkShlycMEY